modules/registrars/synergywholesaledomains/synergywholesaledomains.php

```diff
@@ -2537,7 +2538,7 @@ function synergywholesaledomains_AdminCustomButtonArray(array $params)
         'Push' => 'push',
     ];
 
-    if (substr($params['tld'], -3) == '.au') {
+    if (Str::endsWith($params['tld'], '.au')) {
```

Comment:
I think direct au domains just return 'au' and not '.au', for the other TLD's this works cause it would be 'com.au'

Can we double check what a direct au domain returns as the TLD?